app/modules/mqtt.c

```diff
@@ -873,8 +885,7 @@ static sint8 socket_connect(struct espconn *pesp_conn)
 #ifdef CLIENT_SSL_ENABLE
   if(mud->secure)
   {
-      espconn_secure_set_size(ESPCONN_CLIENT, 5120); /* set SSL buffer size */
```

Comment:
In case of set size for each connection there was some memory leaks... Maybe it fixed, will try to move back and test.